build.gradle

```diff
@@ -104,6 +105,43 @@ switch (JavaVersion.current()) {
         break;
 }
 
+task cloneAndBuildJspecify {
+    description 'Obtain or update JSpecify.'
+    doLast {
+        if (JavaVersion.current() > JavaVersion.VERSION_1_8) {
+            if (file(jspecifyHome).exists()) {
+                exec {
+                    workingDir jspecifyHome
+                    executable 'git'
+                    args = ['pull', '-q']
+                    ignoreExitValue = true
+                }
+            } else {
+                println 'Cloning JSpecify repository.'
+                exec {
+                    workingDir "${jspecifyHome}/../"
+                    executable 'git'
+                    args = [
+                        'clone',
+                        '-q',
+                        '--depth',
+                        '1',
+                        'https://github.com/jspecify/jspecify.git'
+                    ]
+                }
+            }
+            println 'Building JSpecify repository.'
+            exec {
+                println "Executing in directory: ${jspecifyHome}"
+                workingDir jspecifyHome
+                commandLine 'sh', '-c', './gradlew assemble'
+            }
+        }
+    }
+}
+
+test.dependsOn cloneAndBuildJspecify
```

Comment:
Not all tests require JSpecify. Can we target this only to the JSpecify tests?

Reply:
It seems like even if add dependency to NullnessNullMarkedTest, it did not automatically  apply to the Test task